ngrok/README.md

```diff
@@ -18,16 +18,22 @@
 [apache-url]: https://github.com/ngrok/ngrok-rs/blob/main/LICENSE-APACHE
 
 [Website](https://ngrok.com) |
-[API Docs (unstable)](https://ngrok.github.io/ngrok-rs/ngrok)
-
-ngrok is a globally distributed reverse proxy commonly used for quickly getting
-a public URL to a service running inside a private network, such as on your
-local laptop. The ngrok agent is usually deployed inside a private network and
-is used to communicate with the ngrok cloud service.
-
-This is the ngrok agent in library form, suitable for integrating directly into
-Rust applications. This allows you to quickly build ngrok into your application
-with no separate process to manage.
+[API Docs](https://ngrok.github.io/ngrok-rs/ngrok)
+
+ngrok is a simplified API-first ingress-as-a-service that adds connectivity, 
+security, and observability to your apps.
+
+ngrok-rs, our native and idiomatic crate for adding a public internet address 
+with secure ingress traffic directly into your Rust apps 🦀. If you’ve used ngrok in 
+the past, you can think of ngrok-rs as the ngrok agent packaged as a Rust crate.
+
+ngrok-rs lets developers serve Rust services on the internet in a single statement 
+without setting up low-level network primitives like IPs, NAT, certificates, load 
+balancers, and even ports! Applications using ngrok-rs listen on ngrok’s global ingress 
+network using an incoming stream with tokio’s `AsyncRead` and `AsyncWrite` traits — 
+compatible with `axium::Server::builder()` and `hyper::Server::builder()`. This makes 
+it easy to add ngrok-rs into any application that uses axium or hyper — the most 
```

Comment:
* `axum`, not `axium`
* Technically, those are the same type. Axum uses Hyper under the hood and just re-exports its `Server`. The axum support comes from its `Connected` trait, but people probably won't be super familiar with it.